bml/lib/browbeat_run.py

```diff
@@ -93,7 +96,11 @@ def _get_tests(self, workload_search=None,
                    times_search=None,
                    scenario_search=None):
         for index_result in self._elastic_connection:
-            workload = self._map_index_to_workload(index_result['_index'])
```

Comment:
is this left over from when you where trying to handle errors? Do we still need it?